docs/packages/xstate-vue/index.md

```diff
@@ -93,7 +93,7 @@ A [Vue composition function](https://v3.vuejs.org/guide/composition-api-introduc
 
 ### `useActor(actor, getSnapshot)`
 
-A [Vue composition function](https://v3.vuejs.org/guide/composition-api-introduction.html) that subscribes to emitted changes from an existing [actor](https://xstate.js.org/docs/guides/actors.html).
+A [Vue composition function](https://v3.vuejs.org/guide/composition-api-introduction.html) that provides access to an existing [actor](https://xstate.js.org/docs/guides/actors.html). Can be in combination with Vue's [computed](https://vuex.vuejs.org/guide/composition-api.html#accessing-state-and-getters) to subscribe to changes from the actor. 
```

Comment:
Please note that this is just a suggestion, feel free to adjust/reword if needed.
```suggestion
A [Vue composition function](https://vuejs.org/guide/extras/composition-api-faq.html) that provides access to an existing [actor](https://xstate.js.org/docs/guides/actors.html). If you need to listen for different actor instances throughout the lifecycle of this hook then you might need to retain reactivity when destructuring from reactive props or another reactive variable. This can be done using Vue's [computed](https://vuejs.org/api/reactivity-core.html#computed). 
```

1. I would like to put emphasis here that you only need to wrap this if you need to listen for the actor reference being changed - and not if you only want to subscribe to a **single** actor.
2. I would feel better if we'd link to Vue docs here instead of linking to Vuex
3. The note about `computed` could be moved to the bottom of this section, as it's additional info and not necessarily something that we need to start with

Reply:
Good catch on the link. Also you are right about making computed an option rather than a requirement.